LPS25H.class.nut

```diff
@@ -44,9 +55,7 @@ class LPS25H {
 
     // -------------------------------------------------------------------------
     function init() {
-        enable(1);
-        _referencePressure = getReferencePressure();
-        enable(0);
+        softReset();
     }
 
     // -------------------------------------------------------------------------
```

Comment:
We tend to only use static methods in instances where it's intended to be invoked in a static context. Since _twosComp is internal only, I don't think there's much reason to change..